WhaleSpotting/react-app/src/styles/Profile.scss

```
@@ -0,0 +1,133 @@
+@import './Constants.scss';
+
+.body {
+    max-width: 1024px;
+    font-family: $fontFamily;
+    margin: 0 auto 0 auto;
+    display: flex;
+    flex-direction: row;
+}
+
+.profile-pane {
+    height: 100vh;
+    width: 250px;
+    background-color: $secondaryDark;
+}
+
+.feed{
+    padding: 0px 20px;
+    flex: 1;
+}
+
+.outer-container {
+    width: 90%;
+    flex-direction: column;
+}
+
+.little-text{
+    font-size: $bodyFontSize;
+    margin: 0px;
+}
+
+.joined{
+    margin-top:-7px;
+}
+
+.reported{
+    margin-left: 10px;
+}
```

```suggestion
.little-text{
    margin: 0;
    
    &.joined{
        margin-top:-7px;
    }

    &.reported{
        margin-left: 10px;
    }
}
```
I think this can be changed to  this